Samples/word-blazor-add-in/README.md

```diff
@@ -0,0 +1,118 @@
+---
+page_type: sample
+urlFragment: word-blazor-add-in
+products:
+  - office-add-ins
+  - office
+languages:
+  - javascript
+  - C#
+extensions:
+  contentType: samples
+  technologies: Add-ins
+  createdDate: '04/14/2022 10:00:00 PM'
+description: 'Create a Blazor Webassembly Word add-in showcasing some samples.'
+---
+
+# Create a Blazor Webassembly Word add-in
+
+This sample shows how to build a Word add-in using .NET Blazor technologies. Blazor Webassembly allows you to build your addins using .NET, C# and JavaScript to interact with the OfficeJS API. The add-in uses JavaScript to work with the document and Office JS APIs, but you build the user interface and all other non-Office interactions in C# and .NET Core Blazor technologies.
+
+- Work with Blazor Webassembly in the context of Office.
```

Comment:
This bulleted list should have a lead-in sentence such as "This sample shows you the following." Or something like that! But it's also sort of redundant to the preceding paragraph so you could delete it.

Reply:
Fixed in earlier commit